rpc/src/rpc_actor.rs

```diff
@@ -183,4 +184,27 @@ impl Receive<GetFullCurrentHead> for RpcServer {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+impl Receive<GetStatsMemory> for RpcServer {
+    type Msg = RpcServerMsg;
+
+    fn receive(&mut self, ctx: &Context<Self::Msg>, msg: GetStatsMemory, sender: Sender) {
+        if let GetStatsMemory::Request = msg {
+            if let Some(sender) = sender {
+                let me: Option<BasicActorRef> = ctx.myself().into();
```

Comment:
You don't need to specify types here
```rust
let me = ctx.myself().into();
```

Reply:
`error[E0282]: type annotations needed
   --> rpc/src/rpc_actor.rs:195:21
    |
195 |                 let me = ctx.myself().into();
    |                     ^^ consider giving `me` a type`